core/shared/src/main/scala/zio/Schedule.scala

```diff
@@ -71,12 +71,24 @@ object Schedule {
   final def doWhile[A](f: A => Boolean): Schedule[A, A] =
     ZSchedule.doWhile(f)
 
+  /**
+   * See [[ZSchedule.doWhileM]]
+   */
+  final def doWhileM[A](f: A => UIO[Boolean]): Schedule[A, A] =
+    ZSchedule.doWhileM(f)
+
   /**
    * See [[[ZSchedule.doUntil[A](f:* ZSchedule.doUntil]]]
    */
   final def doUntil[A](f: A => Boolean): Schedule[A, A] =
     ZSchedule.doUntil(f)
 
+  /**
+   * See [[ZSchedule.doUntilM]]
+   */
+  final def doUntilM[A](f: A => UIO[Boolean]): Schedule[A, A] =
```

Comment:
The only comment I'd make is that, `U` comes before `W`, but that problem was there before, so 👍 

Thanks for this contribution!